screens/WalletDetailScreen.tsx

```diff
@@ -68,6 +77,26 @@ const IconImage = styled(Image)`
   overflow: hidden;
 `;
 
+const EyeIconImage = styled(Image)`
+  width: 25;
+  height: 25;
+  overflow: hidden;
+  left: 5;
+`;
+
+const VisibilityArea = styled(View)`
+  flex-direction: row;
+  justify-content: flex-end;
+  right: 15;
+  top: 10;
+  position: absolute;
+  align-items: center;
+`;
+
+const VisibilityText = styled(Text)`
```

Comment:
Use the `T` component, not the base `Text` component.  If we ever need to use a new font style, it should be implemented within `T`, or as an extension of T.